login.spec.js

```diff
@@ -0,0 +1,47 @@
+describe('Login', () => {
+  beforeEach(() => {
+    cy.visit('/login');
+  });
+
+  it('should display the login form', () => {
+    cy.get('form').should('be.visible');
+    cy.get('input[type="text"]').should('be.visible');
+    cy.get('input[type="password"]').should('be.visible');
+    cy.get('button[type="submit"]').should('be.visible');
+  });
```

Comment:
The code for entering valid credentials and checking for successful login is repeated in multiple tests. Consider creating a helper function to reduce code duplication.